go.mod

```diff
@@ -107,7 +107,7 @@ require (
 	github.com/rancher/channelserver v0.5.1-0.20220405170618-28c9b37deff1
 	github.com/rancher/dynamiclistener v0.3.4
 	github.com/rancher/eks-operator v1.1.5
-	github.com/rancher/fleet/pkg/apis v0.0.0-20220916041858-eae202f68985
+	github.com/rancher/fleet/pkg/apis v0.0.0-20230116142544-82bfc275d117
```

Comment:
Should this instead point to the v0.5.2-rc1 ([fc276f5](https://github.com/rancher/fleet/commit/fc276f5505bec0be0da9f6560443495586de566e)) commit instead of the master branch?

Reply:
The commit hash is fine. This error happens because we have a module (aka a go.mod file) inside another module. Which is not how modules were designed to be used.